src/rendering/glyphs/TabBendGlyph.ts

```diff
@@ -189,7 +189,7 @@ export class TabBendGlyph extends Glyph {
             case BendType.BendRelease:
                 renderingPoints.push(new TabBendRenderPoint(0, note.bendPoints![0].value));
                 renderingPoints.push(new TabBendRenderPoint((BendPoint.MaxPosition / 2) | 0, note.bendPoints![1].value));
-                renderingPoints.push(new TabBendRenderPoint(BendPoint.MaxPosition, note.bendPoints![3].value));
+                renderingPoints.push(new TabBendRenderPoint(BendPoint.MaxPosition, note.bendPoints!.at(-1)?.value));
```

Comment:
This change worries me a bit. It indicates that various changes might be missing in this PR if this is needed. 

There are many places across the codebase where bends are handled according to the bend type but only 1 was adapted, like: 

* ScoreBendGlyph
* MidiFileGenerator

It would feel a lot safer if we can keep the rule on how many points particular bend types have. Maybe we can rather create the expected amount of points in the MusicXML importer instead of adapting the logic across alphaTab to be fully flexible around point counts.

Reply:
Hi, I agree, that must be a better and safer way to implement this